tests/testthat/test-property.R

```diff
@@ -167,7 +167,8 @@ test_that("properties can be base, S3, S4, R7, or R7 union", {
 
   my_class <- new_class("my_class",
     properties = list(
-      anything = NULL,
+      anything = any_class,
+      null = NULL,
```

Comment:
`missing = missing_class`? for completeness?

Reply:
But then there's no way to create the object 😄